result/flow.yaml

```diff
@@ -45,47 +47,31 @@ components:
               frames_rx_rate:
                 x-field-uid: 7
           x-field-uid: 3
-    Flow.MetricGroup.Request:
+    Flow.MetricTag.Filter:
+      description: |-
+        A container for filtering ingress and/or egress metric tags.
+        The Tx stats may not be applicable in both the request and response filter.
       type: object
       properties:
-        choice:
+        name:
           description: |-
-            The type of disaggregation to be applied to the flow metrics.
+            A metric tag name that MUST exist in a flow packet or
+            flow egress_packet configuration
           type: string
-          default: ingress
           x-field-uid: 1
-          x-enum:
-            ingress:
-              x-field-uid: 1
-            egress:
-              x-field-uid: 2
-        ingress:
+        filters:
           description: |-
-            Disaggregate the flow metrics by ingress packet header field names.
-            If the items is empty then no disaggregation will be done. 
+            A list of filters that can be applied to the metric tag name.
+            By default all values will be included in the flow metric results.
           type: array
-          default: []
           items:
             type: string
-          x-constraint:
-          - "/components/schemas/Flow/properties/packet/../metric_group"
+            format: hex
```

Comment:
Restriction on hex format should be removed. If the entire field is tracked, then the filter value is expected to be in the native format of that field.